alicloud/resource_alicloud_slb_listener.go

```diff
@@ -417,7 +418,7 @@ func resourceAliyunSlbListenerRead(d *schema.ResourceData, meta interface{}) err
 
```

Comment:
Please set the new id after getting new parts.
In addition, please set the new id in the `Update` method.